tgui/packages/tgui/interfaces/MedicalRecords/constants.ts

```diff
@@ -0,0 +1,29 @@
+export const PHYSICALSTATUS2COLOR = {
+  Active: 'good',
+  'Physically Unfit': 'purple',
+  Unconscious: 'average',
+  Deceased: 'bad',
+} as const;
```

Comment:
Why is `'Physically Unift'` in a string, is it because there's two words in it? I'd rather they all be strings, or we could switch the name to simply `Unfit` or perhaps even `Debilitated`. thinking about it i think Debilitated is the best option (medical knowledge :) ), but that's up to you

Reply:
Yes, it's in a string because it contains a space. But I agree that Debilitated has a better ring to it (and that a crewmember physically being Physically Unfit felt a bit redundant), so I'll update it accordingly.